specs/boot_loader_specification.md

```diff
@@ -264,13 +264,18 @@ The following keys are recognized:
 
   Example: `linux /6a9857a393724b7a981ebb5b8495b9ea/3.8.0-2.fc19.x86_64/linux`
 
-* `initrd` specifies the initrd to use when executing the kernel (`cpio` image).
-  The value is a path relative to the root of the file system
-  containing the boot entry snippet itself.
+* `initrd` specifies an initrd to use when executing the kernel (`cpio` image).
+  The value is a path
+  relative to the root of the file system containing the boot entry snippet itself.
   This key may appear more than once,
   in which case all specified images are used,
   in the order they are listed.
 
+  To increase compatibility,
```

Comment:
This begs the question "to increase compatibility with what?", since making this implementation-defined, might lead to more incompatibility between different implementations. Judging from the rationale in the commit message, it's to be more compatible with the semantics of other keys, but that's not clear form the text here alone.

I also think that this should note the preferred behaviour, so that implementations may converge on it.